batch_job/src/batch_file.h

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (C) 2018- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#ifndef BATCH_FILE_H
+#define BATCH_FILE_H
+
+#include "batch_job.h"
+#include "list.h"
+
+typedef enum {
+    BATCH_FILE_INCLUDE,   /* This file is included in files string. (Default) */
+    BATCH_FILE_EXCLUDE,   /* This file is excluded from files string. */
+} batch_file_inclusion_t;
+
+struct batch_file {
+	char *host_name;
```

Comment:
See above first comment. host_name sounds to much like a name of a machine. exe_name sounds to much like the name of an executable.